src/components/DeleteCardPopup.js

```diff
@@ -0,0 +1,28 @@
+// The class extends Popup to inherit basic popup functionality
+import Popup from "./Popup.js";
+
+class DeleteCardPopup extends Popup {
+  constructor({ popupSelector, handleFormSubmit }) {
+    super({ popupSelector });
+    this._handleFormSubmit = handleFormSubmit;
+    this._popup = document.querySelector(popupSelector);
```

Comment:
`    this._popup = document.querySelector(popupSelector);
`

**No** need to search the **modal** again in the children of `Popup` because the parent has already found it in its constructor as `this._popupElement`. It’s available in any child.